bridge_adaptivity/module/models.py

```diff
@@ -109,6 +111,7 @@ class SequenceItem(models.Model):
     activity = models.ForeignKey('Activity', null=True, on_delete=models.CASCADE)
     position = models.PositiveIntegerField(default=1)
     score = models.FloatField(null=True, blank=True, help_text="Grade policy: 'p' (problem's current score).")
+    suffix = models.CharField(max_length=10, default='')
```

Comment:
It is a too common name, please add some comment

Reply:
done